### PR TITLE
Hot fix events controller test

### PR DESCRIPTION
**Pull request by susannasouv**

### CODE REVIEW

**Review by LizzySteger — APPROVED** [review]
looks good to me!

File: spec/controllers/event_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    before :each do
+      @event = Event.create(title: 'event title')
+      get :edit, id: @event.id
+    end
+    it 'properly renders' do
+      expect(response).to render_template(:edit)
+    end
+  end
+
+  describe '#update' do
```
Comment (jeremyjpark) [inline spec/controllers/event_controller_spec.rb:52]: Should there be a testcase where there's no change? Ie, someone wishes to make an update but changes their mind and presses cancel.

Comment (jeremyjpark) [reply spec/controllers/event_controller_spec.rb:52]: Or the case where they update and don't actually change the contents of the event?

Comment (susannasouv) [reply spec/controllers/event_controller_spec.rb:52]: This is a controller method; the "cancel" case is not handled by this method at all.

Comment (susannasouv) [reply spec/controllers/event_controller_spec.rb:52]: bitmoji

**Review by jeremyjpark — APPROVED** [review]

**Pull request merged**